index.html

```diff
@@ -38,6 +39,204 @@ <h2 id="job-title">I am a Software Developer</h2>
+                        </div>
+                        <div class="button-div">
+                            <ul class="button-list">
+                                <li><a href=""><button id="multi-post-button">css</button></a></li>
+                                <li><a href=""><button id="multi-post-button">html</button></a></li>
+                                <li><a href=""><button id="multi-post-button">bootstrap</button></a></li>
+                                <li><a href=""><button id="multi-post-button">Ruby</button></a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <a href=""><button id="first-see-project-button">See Project</button></a>
+                </div>
+                <div class="project-1">
+                    <a id="project-backdrop" href=""><img src="images/Img Placeholder-2.png" alt=""></a>
+                    <div class="floating">
+                        <div>
+                            <h3 id="project-title">Profesional Art Printing Data</h3>
+                        </div>
+                        <div>
+                            <p id="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        </div>
+                        <div>
+                            <a href=""><button class="project-1-buttons">html</button></a>
+                            <a href=""><button class="project-1-buttons" >bootstrap</button></a>
+                            <ahref=""><button class="project-1-buttons">Ruby</button></a>
+                        </div>
+                    </div>
+                    <div class="see-project-div">
+                        <a href=""><button id="see-project-button">See Project</button></a>
+                    </div>
+                </div>
+                <div class="project-1">
+                    <a id="project-backdrop" href=""><img src="images/Img Placeholder-2.png" alt=""></a>
+                    <div class="floating">
+                        <div>
+                            <h3 id="project-title">Profesional Art Printing Data</h3>
+                        </div>
+                        <div>
+                            <p id="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        </div>
+                        <div>
+                            <a href=""><button class="project-1-buttons">html</button></a>
+                            <a href=""><button class="project-1-buttons" >bootstrap</button></a>
+                            <ahref=""><button class="project-1-buttons">Ruby</button></a>
```

Comment:
the tag ahref its wrong, you would put a space between

Reply:
Kindly, whenever you use a link label "to" put the correct link in the "href" attribute, if it does not have a real link, put a "#"